Cargo.toml

```diff
@@ -1,22 +1,23 @@
 [package]
 name = "passage_flex"
-version = "0.1.1"
+version = "0.1.2"
 edition = "2021"
 authors = ["support@passage.id"]
 description = "Provides verification of server-side authentication for applications using Passage Passkey Flex"
 homepage = "https://github.com/passageidentity/passage-flex-rust"
 repository = "https://github.com/passageidentity/passage-flex-rust"
 readme = "README.md"
 license = "MIT"
+include = ["/src", "README.md"]
```

Comment:
This change removes the following unnecessary files from `cargo package --list`:
![Screenshot 2024-08-28 at 10 49 38 AM](https://github.com/user-attachments/assets/98497b47-ef32-465f-a82b-d1262cbfc4b7)